encoding.go

```diff
@@ -163,12 +172,13 @@ func parseEoAccountNode(serialized []byte, depth byte) (VerkleNode, error) {
 func parseSingleSlotNode(serialized []byte, depth byte) (VerkleNode, error) {
 	var values [NodeWidth][]byte
 	offset := leafStemOffset
-	ln := NewLeafNodeWithNoComms(serialized[offset:offset+StemSize], values[:])
+	epochOffset := leafStemOffset + StemSize + 2*banderwagon.UncompressedSize
+	ln := NewLeafNodeWithNoComms(serialized[offset:offset+StemSize], values[:], AccessTimestampFromBytes(serialized[epochOffset:epochOffset+EpochSize]))
```

Comment:
another reason not to do timestamp after all, is that timestamps take more disk space than an epoch, which could be encoded as two bytes for a very, very long time (if an epoch is 6 months, that would be 32000+ years), which is good for disk space since all leaf nodes will have to have an epoch.

With a timestamp, that would mean 4 bytes (as of today) and will need another byte by 2106 (still far out, but not as good)